cmd/snap/cmd_info.go

```diff
@@ -146,6 +146,42 @@ func formatDescr(descr string, max int) string {
 	return strings.TrimSuffix(out.String(), "\n")
 }
 
+func maybePrintCommands(w io.Writer, allApps []client.AppInfo, n int) {
+	if len(allApps) == 0 {
+		return
+	}
+
+	commands := make([]string, 0, len(allApps))
+	for _, app := range allApps {
+		if app.Daemon != "" {
+			continue
+		}
+
+		var cmdStr string
+		if len(app.Aliases) == 0 {
+			cmdStr = app.Name
```

Comment:
if I understood the idea is to use filepath.Base(app.WrapperPath()) instead of app.Name ?